src/views/orb-view.ts

```diff
@@ -346,6 +355,10 @@ export class OrbView<N extends INodeBase, E extends IEdgeBase> implements IOrbVi
   };
 
   zoomed = (event: D3ZoomEvent<any, any>) => {
+    // If zoom is disabled then return
```

Comment:
Btw it wasn't required for the drag because nodes can't be dragged, but the zoom interaction disable/enable should be implemented on the `orb-view-map`. This can be another PR.

Reply:
Sure will raise a new PR related to changes in `orb-view-map`.